### ISSUE TITLE
Airlock docs are completely separate from general output releasing docs

### ISSUE THREAD

**Issue by bloodearnest** [opensafely-core/airlock#712]
For example, airlock docs don't link to https://docs.opensafely.org/outputs/requesting-file-release/, and that doesn't link to the airlock docs.

We should probably at least have links between the two.

There's maybe an argument for embedding the output releasing docs in the airlock docs in some way, so that users can view that from level 4.

Or making docs.opensafely.org accessible from the backend via the proxy, so they can follow the link from L4 VM.

https://bennettoxford.slack.com/archives/C01UJLWNZJT/p1731435121222049

**Comment by LFISHER7** [comment]
I agree that the output releasing docs should be embedded within the airlock docs. 

As a researcher, I want to be able to view those docs as I am creating a request to make sure my disclosure controls meet the requirements.

As an output checker I want to be able to check the disclosure control rules whilst I'm checking, and be able to link to them in comments I am leaving within Airlock
